### PR TITLE
Add CEL tests to TestGrid Dashboard

### PR DESCRIPTION
**Pull request by victoriajyang**
Add CEL conformance tests to TestGrid dashboard

### CODE REVIEW

**Comment by victoriajyang** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
>  **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)
> 
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md). If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

I have CLA access now

File: testgrid/config.yaml [hunk 1]
```diff
@@ -2054,7 +2054,20 @@ test_groups:
...
+ #CEL Conformance Tests
+- name: cel-go-conformance-tests
+  gcs_prefix: cel-conformance/test-logs
+  open_test_template: # The URL template to visit after clicking on a cell
+    url: 
+  file_bug_template: # The URL template to visit when filing a bug
+    url: https://github.com/google/cel-go
+  results_text: See these results on Google Cloud Storage # Text to show in the about menu as a link to another view of the results
+  results_url_template: # The URL template to visit after clicking
+    url: https://pantheon.corp.google.com/storage/browser/cel-conformance/
```
Comment (spiffxp) [inline testgrid/config.yaml]: Can you use a non-`corp.google.com` URI?

**Comment by spiffxp** [comment]
Please squash down to a single commit now that you've got tests happy

**Comment by spiffxp** [comment]
It is not entirely clear to me how these changes relate to Kubernetes?

**Comment by spiffxp** [comment]
/hold
I'm not clear on why/whether these belong here?

**Comment by victoriajyang** [comment]
> /hold
> I'm not clear on why/whether these belong here?

Originally I was going to make our own TestGrid instance but @michelle192837 said I could add it to the existing one.

**Comment by michelle192837** [comment]
Yup, the current policy I believe is if you're okay with your results being public and on the K8s TestGrid, then feel free to add your results here.

**Comment by michelle192837** [comment]
/lgtm

But leaving the hold on for @spiffxp's opinion.

**Comment by spiffxp** [comment]
/approve
/hold cancel
I'm not entirely sure I agree with the longevity of putting things unrelated to k8s on testgrid.k8s.io, but I won't block given that I consider @michelle192837 to have way more ownership of testgrid than I

**Comment by michelle192837** [comment]
Yeah, it shouldn't be the case forever, but for now think this is the best we've got. Consider it blocked on #10409; it's this or some impossible/really expensive work.

**Pull request merged**